app/docs/markdown/getting-started-with-dockstore.md

```diff
@@ -93,6 +93,31 @@ Press the 'Add Tag' button to begin creating tags for the different versions of
 
 The fields in the form should correspond to the actual values on GitHub/Bitbucket/GitLab and Quay.io/Docker Hub in order for the information to be useful to other users. Selecting `Hidden` will prevent the tag from appearing in the public listing of tags for the image.
 
+## [Different Ways To Register Tools on Dockstore](#differet-ways-to-register)
```

Comment:
differet-ways-to-register should be different-ways-to-register

This should not really be in the getting started since this is will be for pretty advanced developers. 
Maybe put this in advanced features or start a new document section for developers. 

This looks good in general though